endpoints/events/event.go

```diff
@@ -188,7 +189,12 @@ func ParseEventRequest(r *http.Request) (*analytics.EventRequest, []error) {
 	}
 
 	// Bidder
-	event.Bidder = r.URL.Query().Get(BidderParameter)
+	bidderName := r.URL.Query().Get(BidderParameter)
+	if normalisedBidderName, ok := openrtb_ext.NormalizeBidderName(bidderName); ok {
+		bidderName = normalisedBidderName.String()
+	}
```

Comment:
should we error out if no normalized bidder name found

Reply:
I was thinking of passing it through the bidderName if we cannot normalise the bidder name which is inline with current flow.

Reply:
Does downstream code assume the bidder name has been normalized? If so then we are breaking that assumption if the bidder name is not found.

Reply:
Here is the workflow that I understood from the code - 
- While preparing the seatbid response, PBS modify bids to add event URL in seats. The [modifyBidsForEvents](https://github.com/prebid/prebid-server/blob/31b4c2051adc996ec25945fdf681cd42bc43baed/exchange/events.go#L48-L59) function prepares the /event endpoint that will eventually get called.
- If we notice the modifyBidsForEvents method also passes bidderName and that is the bidderName in seat which is normalised because that bidderName is used to lookup bidderInfos in [isModifyingVASTXMLAllowed method.](https://github.com/prebid/prebid-server/blob/31b4c2051adc996ec25945fdf681cd42bc43baed/exchange/events.go#L61-L64)
- Now since the same normalised bidder name is also being used to prepare the event endpoint that means that we can do normalisation of the bidder name in the incoming /event endpoint.

Let me know if any more questions.